README.md

```diff
@@ -436,6 +437,10 @@ The inline attributes decides whether the input fields and selected tags will be
 #### allowUnique (optional, defaults to `true`)
 This prop controls whether tags should be unique.
 
+<a name="allowDragDrop"></a>
+#### allowDragDrop (optional, defaults to `true`)
+This prop controls whether tags should be allowed to dragged & dropped.
```

Comment:
same as above ? `This prop controls whether tags should have drag and drop features enabled.`